src/guessWords/containers/App.js

```diff
@@ -0,0 +1,62 @@
+import React, { useState, useEffect, useCallback } from "react";
+import Congrats from "../components/Congrats";
+import GuessedWords from "../components/GuessedWords";
+import Input from "../containers/Input";
+
+function App() {
+  const [success, setSuccess] = useState(false);
+  const [secretWord, setSecretWord] = useState("");
+  const [guessedWords, setGuessedWords] = useState([]);
+  const [matchcount, setMatchCount] = useState([]);
+  const guesswordCallback = useCallback(
+    (guessedWord) => {
+      if (guessedWord == secretWord) {
+        setSuccess(true);
```

Comment:
a simple way that doesn't need `if` statement.
```suggestion
        setSuccess(guessedWord === secretWord);
```